### DISCUSSION TITLE
Filter `after` task to `alias` selector?

### DISCUSSION THREAD

**Discussion by leevigraham** [deployphp/deployer#3733] [Help needed] [answered]
I want to:

```after('deploy:symlink', 'deploy:sentry');```

but only on production.

Is there a way to do this with selectors?

I can do something like 

```
after('deploy:symlink', function(){
    $alias = currentHost()->getAlias();
    if($alias != 'production') {
        return;
    }
    invoke('deploy:sentry');
});
```

**Comment by leevigraham** [comment, marked as answer]
I think this also works… adding a selector to the task:

```task('deploy:sentry')->select('production');```

Shows the correct plan:

![image](https://github.com/deployphp/deployer/assets/25124/a7dbc23c-3a5d-4e0a-befe-d9a5301ee4fb)

**Reply by antonmedv** [reply]
Yes, this is a correct answer.
